webpack.config.js

```diff
@@ -9,6 +9,7 @@
 
 const path = require('path')
 const webpack = require('webpack')
+const { ESBuildMinifyPlugin } = require('esbuild-loader')
 const fs = require('fs')
 const TerserPlugin = require('terser-webpack-plugin')
```

Comment:
can this= be removed ?

Reply:
Yeah should be able to remove it, I don't see it being used elsewhere.